contracts/smart-account/interfaces/IAccountRecoveryModule.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+interface IAccountRecoveryModule {
```

Comment:
what are your thoughts about declaring all public and external functions in the interface itself?

Reply:
I don't know if this is needed in this exact case. I don't expect people interact a lot with our Account Recovery through the interface or inherit the interface in other contracts.
wdyt?

Reply:
yeah, I don't think it's necessary but it's just another good-to-follow practice. I did this for all other contracts for the last refactor I merged in develop

Reply:
made it to preserve the consistency 🤝

Reply:
Thank you!